### ISSUE TITLE
A11y component

### ISSUE THREAD

**Issue by raunofreiberg** [reach/reach-ui#567]
### Current Behavior

Finding violations of accessibility is not very straightforward.

### Desired Behavior

Finding violations of accessibility should be as easy as using a component, and visual.

### Suggested Solution

Expose a React component from `@reach/a11y` which would wrap component trees to reveal accessibility violations.

I have a POC implementation available in a [sandbox.](https://codesandbox.io/s/icy-pond-rz3lz?file=/src/App.js)

### Who does this impact? Who is this for?

All users of Reach UI.

### Describe alternatives you've considered

No alternatives were considered.

**Comment by chancestrickland** [comment]
Super cool concept, but I'm not sure if Reach is the right place for it. Could be an interesting proposal for the `axe-core` team since it's extension of that product, or it'd even make a great standalone package. Happy to collaborate on it with you however you decide to approach it!

**Comment by raunofreiberg** [comment]
Me and @chancestrickland made a spin-off show based on this issue. Tune in at: https://github.com/raunofreiberg/axe-mode
